tests/Unit/IO/Test_VolumeData.py

```diff
@@ -15,25 +15,88 @@ class TestIOH5VolumeData(unittest.TestCase):
     def setUp(self):
         # The First tests involve inserting vol files, the h5 file for
         # these will be deleted and recreated for each test as needed.
-        self.file_name_w = os.path.join(Informer.unit_test_path(),
-                                        "IO/pythontest.h5")
-        # The other tests require external data, so we use the file
-        # `VolTestData.h5` which contains data output from the spectre
-        # EvolvePlaneWave3D executable.  However, the file does not contain the
-        # source tree (for file size reasons), so it should not be used as an
-        # example output for any other reason than testing VolumeData.
-        self.file_name_r = os.path.join(Informer.unit_test_path(),
-                                        "IO/VolTestData.h5")
-        if os.path.isfile(self.file_name_w):
-            os.remove(self.file_name_w)
+        # The other tests use a volume data file written using
+        # the write_volume_data() function
+        self.file_name = os.path.join(Informer.unit_test_path(),
+                                      "IO/pythontest.h5")
+
+        if os.path.isfile(self.file_name):
+            os.remove(self.file_name)
+
+        h5_file = spectre_h5.H5File(file_name=self.file_name,
+                                    append_to_file=True)
+        self.tensor_components_and_coords = [
+            ds.DataVector([8.9, 7.6, 3.9, 2.1, 18.9, 17.6, 13.9, 12.1]),
+            ds.DataVector([0.0, 1.0, 0.0, 1.0, 0.0, 1.0, 0.0, 1.0]),
+            ds.DataVector([0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 1.0, 1.0]),
+            ds.DataVector([0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0]),
+            ds.DataVector([-78.9, -7.6, -1.9, 1.9, 6.3, 1.7, 9.8, 1.2]),
+            ds.DataVector([-7.9, 17.6, 1.9, -8.1, -6.3, 21.7, 6.8, -0.2]),
+            ds.DataVector([1.0, 6.6, 28.0, -8.1, -26.3, 32.7, 26.8, -3.2])
+        ]
```

Comment:
Since I think you don't care about these particular values, how about this:

```suggestion
        self.tensor_components = np.random.rand(4, 8)
```

Then you have 4 rows with 8 random values each that you can use as sample data. You can convert a row to a DataVector like `ds.DataVector(self.tensor_components[0])` where you need to (e.g. to construct the `ds.TensorComponent`s below).